content/cpp/concepts/math-functions/terms/asinh/asinh.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'asinh'
+Description: 'Returns the inverse hyperbolic sine of an angle given in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns the inverse hyperbolic sine of an angle given in radians.
+
+## Syntax
+
+```cpp
+asinh(n)
+```
+
+The `cmath` header file must be included to use this function.
```

Comment:
```suggestion
The `n` parameter can be a floating-point value (e.g., a `long` or `double`) or an integral type (e.g., an `int` or `bool`).
```

I think that mentioning the `cmath` header isn't necessary here since this point is stated in the parent "Math Functions" entry. Including info about the parameter(s) is more relevant to this section.